packages/ember-htmlbars/lib/env.js

```diff
@@ -35,3 +33,5 @@ export default {
 
   helpers: helpers
 };
+
+export var domHelper = domHelper;
```

Comment:
just be careful with `export function Blah() { }`; while valid es6, we do not yet have a version of the es6 module transpiler in ember that supports this syntax :(